### ISSUE TITLE
Validation loss keeps increasing on training

### ISSUE THREAD

**Issue by RolandCita** [ZQPei/deep_sort_pytorch#177]
Hello, 

I'm trying to train a custom dataset with this model, and whatever i try, the validation loss is always high. Even with MARS Dataset.
![image](https://user-images.githubusercontent.com/52562823/99701124-b4b5a500-2a9c-11eb-8731-7021eca2723a.png)
This here would be on MARS Dataset, i know it's 6 epochs in, but it doesn't look like it would get somewhere.

![image](https://user-images.githubusercontent.com/52562823/99701217-d57dfa80-2a9c-11eb-94cf-ceb98351dcd2.png)
And this here is on my own dataset.

**Comment by joaquinCaceres** [comment]
I have the same problem. I am training with MARS dataset but in epoch 20 I only get 20% accuracy on the test set.  Has the problem been found?

**Comment by ananthu-aniraj** [comment]
You need to split the train set of the MARS dataset/any custom dataset you are using into "train" and "val" sets with a split of  (2N/3, N/3), where N is the number of images in your training set.
The reason why you need to do this is that we are training an image classifier. So the identities on train and val should be the same.
During inference of this model (for tracking), the softmax operation at the end of the image classifier is removed, so that the model generates embeddings for each object detected by your detector model
